src/GovernorCountingFractional.sol

```diff
@@ -185,8 +218,9 @@ abstract contract GovernorCountingFractional is Governor {
     uint256 constant internal _VOTEMASK = 0xffffffffffffffffffffffffffffffff; // 128 bits of 0's, 128 bits of 1's
 
     /**
-     * @dev Decodes three packed uint128's. Uses assembly because of Solidity language limitation which prevents
-     * slicing bytes stored in memory, rather than calldata.
+     * @dev Decodes three packed uint128's. Uses assembly because of Solidity
```

Comment:
Nit: "because of **a** Solidity language limitation"

Reply:
Good catch, done in d301ac2